catalog_tools/analysis/estimate_beta.py

```diff
@@ -4,8 +4,8 @@
 import numpy as np
 
 
-def estimate_beta_tinti(magnitudes: np.ndarray, mc: float, delta_m: float = 0,
-                        weights: list = None) -> float:
```

Comment:
why are you removing the option for weights? I like to have weights :)

Reply:
Ups, I thought they were useless, but if you like them they will stay!